tests/test_compute_service.py

```diff
@@ -391,6 +391,8 @@ def cleanup_router(router, gateway):
                             % fip.public_ip)
                     test_inst.refresh()
                     test_inst.reboot()
+                    test_inst.stop()
```

Comment:
It would be ideal to add separate tests for this, and include something like checking the state of the instance and waiting until it is in the desired state.
Separating it will also allow us to more easily make some providers skip that test, given that the function is not added for all providers yet

Reply:
updated